sunpy/data/_sample.py

```diff
@@ -116,6 +119,9 @@ def download_sample_data(overwrite=False):
 
         if not results.errors:
             return results
+        else:
+            log.debug('Failed to download file' + str(err.filepath_partial) +
```

Comment:
Should this be info to match the one above or should log above be debug?

Reply:
Since we wish to log to std output, we could keep it 'info' for consistency. Though this log will be useful for debugging purposes as well.